### ISSUE TITLE
Documentation is invalid for 10 LTS

### ISSUE THREAD

**Issue by arnekolja** [WebentwicklerAt/typo3-emogrifier#17]
As of https://docs.typo3.org/c/typo3/cms-core/master/en-us/Changelog/9.5/Deprecation-85970-FileContentObject.html the FILE content object got deprecated in 9 LTS and removed in 10 LTS. The documentation should adapt to that and show an example without this TS part.

As there's no standard way to load file content that's not an image or a template any more - maybe the file path should be a simple setting and loading should be performed inside its own logic?

**Comment by msollmann** [comment]
Yes, the official documentation is invalid, but have a look at EXT:emogrifier/Documentation/Examples/ContentObject/Index.rst. You only have to replace FILE by FLUIDTEMPLATE :-)

**Comment by oliverschloebe** [comment]
> Yes, the official documentation is invalid, but have a look at EXT:emogrifier/Documentation/Examples/ContentObject/Index.rst. You only have to replace FILE by FLUIDTEMPLATE :-)

While this might be working for the template files it still does not for the CSS files.

Any update or solution for this?

Thanks!

**Comment by klodeckl** [comment]
Thanks for the hint. It does also work for CSS files.
